src/index.js

```diff
@@ -24,17 +24,17 @@ const Cell = styled('div').attrs({
 
 Cell.displayName = 'Cell';
 
-const Row = styled('div').attrs({
+const Row = styled('div').attrs(() => ({
   className: 'sticky-table-row'
-})`
+}))`
   display: table-row;
 `;
 
 Row.displayName = 'Row';
 
-const Wrapper = styled('div').attrs({
+const Wrapper = styled('div').attrs(() => ({
```

Comment:
what's the reason for this change?

Reply:
It was deprecated on `styled-components@4` but was backwards compatible - and dropped in v5 - it's point 7 in their FAQ here https://styled-components.com/docs/faqs#what-do-i-need-to-do-to-migrate-to-v4